### PR TITLE
Add base for status view

### PR DESCRIPTION
**Pull request by mzlarsson**
Added base for status view with some mocked values

### CODE REVIEW

File: src/tsx/statusView/statusEffect.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    return (
+      <div className={style.effectContainer}>
+        <EffectContent effect={this.props.effect} />
+      </div>
+    );
+  }
+}
+
+function EffectContent(props: IProps) {
+  switch (props.effect.type) {
```
Comment (AntonNeld) [inline src/tsx/statusView/statusEffect.tsx:21]: I don't remember what we said about inheritance vs composition here, i.e. whether this component should use different logic depending on the effect type or (for example) just always draw a progress bar if the effect has a duration. Do you remember?

I guess we will know which approach is better once we implement more effect types though, so we could always do it like this now and refactor later if we change our mind,

File: src/tsx/coreConnection.tsx [hunk 2]
```diff
@@ -82,6 +105,7 @@ class DummyCoreConnection extends EventTarget implements ICoreConnection {
     this.dispatchEvent(
       new CustomEvent(eventNames.script, { detail: this.script }),
     );
+    this.sendEffectsChangedEvent();
```
Comment (AntonNeld) [inline src/tsx/coreConnection.tsx:108]: The name 'sendEffectsChangedEvent' is not very accurate in this context. Should it be just 'sendEffectsEvent'?

File: src/tsx/statusView/statusEffect.tsx (hunk 1 above)
Comment (mzlarsson) [reply src/tsx/statusView/statusEffect.tsx:21]: I think we might fix with inheritance later. Components may look rather different even if they have the same feature (e.g. video preview will include progress bar whilst audio may need to do a custom one).

**Review by AntonNeld — APPROVED** [review]
LGTM

**Pull request merged**